src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -4060,6 +4060,290 @@ System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator()
                 Diagnostic(ErrorCode.WRN_NullAsNonNullable, "null").WithLocation(10, 30));
```

Comment:
Not related to this PR: Why do we have a dedicated warning for `null` literal here, but a common warning for `null` or possibly null (`WRN_ConvertingNullableToNonNullable`) in `PassingParameters_01` above?

Reply:
`WRN_ConvertingNullableToNonNullable` is the W warning. In `PassingParameters_01`, that warning is reported when assigning a value returned from an another method. In this case, the `null` value is passed (from this method) as an argument to a non-nullable parameter.

---
In reply to: [182836475](https://github.com/dotnet/roslyn/pull/26107#discussion_r182836475) [](ancestors = 182836475)